### ISSUE TITLE
General Setting: Do not apply layouts to new windows automatically

### ISSUE THREAD

**Issue by cbtum** [ianyh/Amethyst#1071]
**Is your feature request related to a problem? Please describe.**
Whenever a new windows comes up on my screen, Amethyst automatically tries to restore the current layout (e.g. column, fullscreen etc). This lead can to many different undesired effects:

- Depending on the size and the content of the new window, such a layout might not make any sense as described here https://github.com/ianyh/Amethyst/issues/1070
- In some cases, I just want to open a new window on top without destroying the previous layout
- If an app is showing some (actually small) popup and I am using Amethyst in fullscreen mode, the popup might end up being fullscreen as well which might both, hide what is behind the popup + might require very long ways if I have to use my mouse
- In some cases, I am not really bothered about the layout and hence rather distracted by Amethyst.

**Describe the solution you'd like**
I would prefer Amethyst to do nothing, unless I press a shortcut. It's very easy/fast to use shortcuts, I still have the same benefits without the undesired effects described above.

**Describe alternatives you've considered**
Amethyst would need to be much more context sensive in the way when and how layouts are applied. I think this is unrealistic.

So in summary, I really like Amethyst but I would prefer to tell Amethyst when I want a to change a layout.

**Comment by ianyh** [comment]
You can configure floating preferences so that you float everything by default.
